samples/snippets/src/main/java/dialogflow/cx/UpdateIntent.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package dialogflow.cx;
+
+import com.google.cloud.dialogflow.cx.v3.Intent;
+import com.google.cloud.dialogflow.cx.v3.Intent.Builder;
+import com.google.cloud.dialogflow.cx.v3.IntentsClient;
+import com.google.cloud.dialogflow.cx.v3.UpdateIntentRequest;
+import com.google.protobuf.FieldMask;
+import java.io.IOException;
+
+public class UpdateIntent {
```

Comment:
Please see https://g3doc.corp.google.com/company/teams/cloud-devrel/dpe/samples/snippets.md?cl=head#rubric for adding appropriate comments